index.html

```diff
@@ -1,271 +1,282 @@
+  
+  
+        <div class="description">
+  
+          <h3 class="project-title">Multi-Post Stories</h3>
+          <p class="project-infor"> A daily selection of privately personalized reads no accounts or sign-ups
+            required. has been the industry standard dummy text ever since the 1500s, when an unknown printer took
+            standard dummy text.
           </p>
-
+  
           <ul class="used-tools">
             <li class="project-list">css</li>
-
             <li class="project-list">html</li>
-
             <li class="project-list">bootstrap</li>
-
             <li class="project-list">Ruby</li>
           </ul>
-
           <button type="button" class="see-prj">See Project</button>
-        </article>
-      </section>
-    </main>
-    <footer class="footer">
-      <div class="contact-us-format">
-        <h1 class="contact-us">CONTACT US</h1>
+        </div>
+  </section>
+
+</main>
+
+
+
+ <footer class="footer">
 
-        <p class="offer-help">
+    <section class="contact-section">
+
+      <div class="contact-instructions">
+        <h2 class="contact-us">CONTACT US</h2>
+         
+        <p class="we-help">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
       </div>
+  
+      <form action="https://formspree.io/f/xjvlllpk" method="POST" class="form">
 
-     <form  action="https://formspree.io/f/xjvlllpk"  method="POST" class="contact" >
-        <input
-          type="text"
-          id="name"
-          name="name"
-          maxlength="30"
-          placeholder="Text input"
-        required/>
+        <input type="text" class="name1" id="name1" name="name" maxlength="30" placeholder="Text input" required />
 
-        <input class="email" type="email" id="email" name="email" placeholder="jeremy@gmail.com" required/>
+  
+        <input class="email1" type="email" id="email1" name="email" placeholder="jeremy@gmail.com" required />
 
-        <textarea
-          class="message"
-          name="message"
-          id="message"
-          maxlength="500"
-          placeholder="Enter text here"
-          required></textarea>
+  
+        <textarea class="textarea1" name="message" id="textarea1" maxlength="500" placeholder="Enter text here" required></textarea>
 
+  
         <button id="see-project" class="see-prj" type="submit">See Project</button>
-      </form> 
-
-      <hr class="below-form">
+ 
+      </form>
+    
+    </section>
 
-      <!--social media icons-->
-      <div class="social">
+                  <!--vector line-->
+     <div class="short">
+        <img class="horizontal-line" src="./background-shapes/mobile-version/Vector 4.png" alt="horizontal-line">
+      </div>
+  
+  
+    <!--social media icons-->
+  
+    <div class="bottom-footer">
+      <div class="social footer-icons">
```

Comment:
Your icons don't quite look like the ones from the template design.

Your design:

<img width="108" alt="Screen Shot 2022-06-07 at 12 29 55" src="https://user-images.githubusercontent.com/82426682/172445896-9ec3ae69-a03f-430e-8a73-e49fe960b6a2.png">

Template design:

<img width="116" alt="Screen Shot 2022-06-07 at 12 30 19" src="https://user-images.githubusercontent.com/82426682/172445947-4633c1aa-5272-4fbe-8686-96b2e832718c.png">

I recommend you download them directly from the Figma template. Please fix this as it is a project requirement.

<img width="894" alt="Screen Shot 2022-06-07 at 12 30 56" src="https://user-images.githubusercontent.com/82426682/172446053-44b392cf-f9d9-4521-9a01-9c8709b82b96.png">